Description: remove whirlpool usage.  the algorithm is missing on at least
 0.9.8g and 0.9.8i, even though there's an OBJ_whirlpool definition in 0.9.8i.
Origin: upstream, http://websvn.kde.org/?view=revision&revision=864423

--- qca-ossl/qca-ossl.cpp
+++ qca-ossl/qca-ossl.cpp
@@ -6616,9 +6616,11 @@
 #ifdef SHA512_DIGEST_LENGTH
 	list += "sha512";
 #endif
+/*
 #ifdef OBJ_whirlpool
 	list += "whirlpool";
 #endif
+*/
 	return list;
 }
 
@@ -6863,10 +6865,12 @@
 		else if ( type == "sha512" )
 			return new opensslHashContext( EVP_sha512(), this, type);
 #endif
+/*
 #ifdef OBJ_whirlpool
 		else if ( type == "whirlpool" )
 			return new opensslHashContext( EVP_whirlpool(), this, type);
 #endif
+*/
 		else if ( type == "pbkdf1(sha1)" )
 			return new opensslPbkdf1Context( EVP_sha1(), this, type );
 		else if ( type == "pbkdf1(md2)" )
